src/win32_config.h

```diff
@@ -31,10 +31,12 @@
  */
 #pragma once
 
+#ifndef WITHOUT_WIN32_CONFIG
```

Comment:
Maybe it is better to do `#ifndef WITH_SSL` and so on, that would allow non-CMake users to also override these settings on Win32.